src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -54,6 +54,18 @@ export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
       },
     };
   }
+  uischema.elements = removeUIElementWithName(
+    passwordFieldName,
+    uischema.elements,
+  );
+
+  const passwordWithConfirmationElement: PasswordWithConfirmationElement = {
+    type: 'PasswordWithConfirmation',
+    label: loc('oie.password.confirmPasswordLabel', 'login'),
```

Comment:
not a required change: but wondering if we ever have an instance where we need the flexibility to pass the label for this element type. if not, we can consider encapsulating this label

Reply:
Do you mean pass the `label` as a prop to `options` instead of here? 

I can do that, I just re-used the `label` attr that was available for the confirm password field to prevent duplicating props in the schema.

Reply:
I mean if `label` can live hardcoded inside the `PasswordWithConfirmation` component

Reply:
Ohhh, the direction that was told to me was that the components should not use `loc` or any translation. That the components should always receive translated text.